inference-engine/src/mkldnn_plugin/nodes/psroi.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Seems like redundant empty line

Reply:
Fixed in separate PR:
https://github.com/openvinotoolkit/openvino/pull/3166